src/cr/cube/matrix/assembler.py

```diff
@@ -1102,7 +1103,21 @@ def _element_values(self):
 
     @lazyproperty
     def _marginal(self):
-        raise ValueError("Not yet implemented")
+        """Marginal object providing values for sort."""
+        propname_by_marginal = {
+            MARGINAL.SCALE_MEAN: "rows_scale_mean",
+            MARGINAL.SCALE_MEAN_STDDEV: "rows_scale_mean_stddev",
+            MARGINAL.SCALE_MEDIAN: "rows_scale_median",
+        }
+        marginal = self._order_spec.marginal
+        marginal_propname = propname_by_marginal.get(marginal)
```

Comment:
It's a fine point, but this can be condensed to:

```python
    marginal = self._order_spec.marginal
    marginal_propname =  {
        MARGINAL.SCALE_MEAN: "rows_scale_mean",
        MARGINAL.SCALE_MEAN_STDDEV: "rows_scale_mean_stddev",
        MARGINAL.SCALE_MEDIAN: "rows_scale_median",
    }.get(marginal)
```

That's one less temporary variable to think of a name for and the reader to keep track of.